src/EnergyPlus/MixerComponent.cc

```diff
@@ -683,6 +683,34 @@ namespace MixerComponent {
         }
     }
 
+    int getZoneMixerIndexFromInletNode(int const &InNodeNum)
```

Comment:
New functions to get mixer or return plenum index.

Reply:
These can be converted to bool functions but are still needed to verify connection is correct.